.github/workflows/inductor-perf-test-nightly-B200.yml

```diff
@@ -0,0 +1,173 @@
+name: inductor-perf-nightly-B200
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/inductor-perf-test-nightly-B200.yml
+  schedule:
+    - cron: 15 0,4,8,12,16,20 * * 1-6
```

Comment:
Let just run this daily instead of every 4 hours unless there is more runners coming?